Imabulary/Imabulary
# [Feature] Prepare mobile app for publishing to AppStore (IMB-163)

## Overview

This PR will prepare app to be published to AppStore by adding Apple sign-in method and temporarily disabling payment system 

## Test cases
1. Run the app on the IOS
2. Login using Apple
3. Verify whether there are no errors, every part of the app operates as expected

## Issue

closes [IMB-163](https://www.notion.so/Prepare-mobile-app-for-publishing-to-AppStore-ba0d41af0cdf4abead61cfb9410317ea?pvs=4)
